tests/RobotFramework/tests/cumulocity/monitoring/monitor_device_collectd.robot

```diff
@@ -0,0 +1,33 @@
+*** Settings ***
+Resource        ../../../resources/common.resource
+Library         ThinEdgeIO
+Library         Cumulocity
+
+Suite Setup     Custom Setup
+Test Teardown    Get Logs
+
+Force Tags      theme:monitoring    theme:c8y    theme:collectd
+
+*** Test Cases ***
+Check running collectd
+    Service Should Be Running    collectd
+    
+Is collectd publishing MQTT messages?
+    ${messages}=    Should Have MQTT Messages    topic=collectd/#    minimum=1    maximum=None
+    # Should Contain    ${messages[0]}    "pid":${pid}
+    # Should Contain    ${messages[0]}    "status":"up"
+
+Check thin-edge monitoring
+    Execute Command    sudo systemctl enable tedge-mapper-collectd
+    Execute Command    sudo systemctl start tedge-mapper-collectd
+    ${tedge_messages}=    Should Have MQTT Messages    topic=tedge/measurements    minimum=1    maximum=None
+    ${c8y_messages}=    Should Have MQTT Messages    topic=c8y/#    minimum=1    maximum=None
```

Comment:
It would have been better if the presence of the expected metrics(`cpu`, `memory` etc) in the JSON could also be validated.

Reply:
@rina23q  Thanks, I forgot to ask you about the right topic, it was my intention to use the "more concrete"